hellabot.go

```diff
@@ -34,6 +34,12 @@ type Bot struct {
 	// Unix domain socket address for other Unixes
 	unixsock string
 	unixlist net.Listener
+	// SASL handler
+	saslCreds struct {
+		user *string
+		pass *string
+	}
```

Comment:
What are these being saved for?

Reply:
Because SASLAuthenticate can be called with different arguments if the user wants so and we need to reference them in our trigger.